public/index.html

```diff
@@ -0,0 +1,281 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Webchat</title>
+    <link rel="icon" href="./favicon.png" />
+    <link rel="stylesheet" href="style.css" />
+    <script src="http://localhost:3000/socket.io/socket.io.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/moment.min.js"></script>
+  </head>
+  <body>
+    <main>
+      <form>
+        <fieldset>
+          <legend>Webchat</legend>
+          <label for="nickname">
+            <input
+              data-testid="nickname-box"
+              id="nickname"
+              name="nickname"
+              type="text"
+              placeholder="nickname"
+              autofocus
+              minlength="1"
+              maxlength="8"
+              size="8"
+              title="O nick pode conter até oito caracteres."
+            />
+          </label>
+          <button
+            data-testid="nickname-save"
+            id="nickBtn"
+            type="button"
+            class="true"
+          >
+            select
+          </button>
+          <span id="chosenNick"></span>
+          <section>
+            <ul id="firstUser"></ul>
+            <ul id="onlineUser"></ul>
+          </section>
+          <label for="messageBox"
+            >message
+            <input
+              data-testid="message-box"
+              id="messageBox"
+              name="messageBox"
+              type="text"
+              placeholder="type your message"
+            />
+          </label>
+          <button
+            data-testid="send-button"
+            id="msgBtn"
+            type="button"
+            class="true"
+          >
+            send
+          </button>
+          <button data-testid="public" type="button" id="public" class="true">
+            Public Chat
+          </button>
+        </fieldset>
+        <section class="messages">
+          <ul id="messagesList" style="list-style: none"></ul>
+          <ul id="privateMessages"></ul>
+        </section>
+      </form>
+    </main>
+    <script>
+      const socket = io('http://localhost:3000');
+
+      const nickName = document.querySelector('#nickname');
+      const nickBtn = document.querySelector('#nickBtn');
+      const chosenNick = document.querySelector('#chosenNick');
+      const messageBox = document.querySelector('#messageBox');
+      const msgBtn = document.querySelector('#msgBtn');
+      const msgList = document.querySelector('#messagesList');
+      const privateMsgList = document.querySelector('#privateMessages');
+      const publicChatBtn = document.querySelector('#public');
+      const privateChatBtn = document.querySelector('#private');
+      const firstUser = document.querySelector('#firstUser');
```

Comment:
@llsxavier essa prática de declarar variáveis no topo de um contexto não é recomendável. Uma alternativa melhor é declarar cada variável o mais próximo possível de onde ela será utilizada. 

Isso facilita a leitura do código e também sua manutenção no futuro :)

Reply:
@vbuxbaum obrigado pelo feedback. Farei a correção no código porque não tinha me atentado a isso anteriormente.
```suggestion
      const nickName = document.querySelector('#nickname');
      const nickBtn = document.querySelector('#nickBtn');
      const chosenNick = document.querySelector('#chosenNick');
      const messageBox = document.querySelector('#messageBox');
      const msgBtn = document.querySelector('#msgBtn');
      const msgList = document.querySelector('#messagesList');
      const privateMsgList = document.querySelector('#privateMessages');
      const publicChatBtn = document.querySelector('#public');
      const privateChatBtn = document.querySelector('#private');
      const firstUser = document.querySelector('#firstUser');
      // comentário teste
```